presets/anime.json

```
@@ -41,7 +41,7 @@
     ],
     "default_aspect_ratio": "896*1152",
     "checkpoint_downloads": {
-        "bluePencilXL_v050.safetensors": "https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/bluePencilXL_v050.safetensors",
+        "bluePencilXL_v050.safetensors": "https://huggingface.co/bluepen5805/blue_pencil-XL/raw/main/blue_pencil-XL-v2.0.0.safetensors",
```

please also adjust the name here to prevent errors and indefinite model downloading:
```suggestion
        "blue_pencil-XL-v2.0.0.safetensors": "https://huggingface.co/bluepen5805/blue_pencil-XL/raw/main/blue_pencil-XL-v2.0.0.safetensors",
```